docs/learn/learn-collator.md

```diff
@@ -8,70 +8,33 @@ description: Understand the role of collators in Polkadot, including block produ
-guarantees because the relay chain provides those. If a parachain block is invalid, it will get
-rejected by validators. The validators are required to check the validity of submitted candidates,
-followed by issuing and collecting statements about the validity of candidates to other validators.
-This process is known as **candidate backing**. Validators receive an arbitrary number of parachain
-candidates with associated PoV from untrusted collators. A candidate is considered _backable_ when
-at least 2/3 of all assigned validators have issued a valid statement about that candidate.
+### Rollup Maintainers
 
-The validator must successfully verify the following conditions in the following order:
-
-1. The candidate does not exceed any parameters in the persisted validation data.
+Collators maintain rollup chains (parachains) by collecting user transactions and producing state transition proofs for Polkadot's validators. In other words, collators aggregate transactions into parachain block candidates and produce state transition
+proofs (Proof-of-Validity, PoV) for validators.
 
-2. The signature of the collator is valid.
+Collators maintain a full node for the relay chain and a full node for their particular parachain, meaning they retain all necessary information to author new parablocks and execute transactions like miners do on PoW blockchains. Under normal circumstances, they
+will collate and execute transactions to create an unsealed block and provide it with a PoV to one or more validators on a Polkadot core.
 
-3. Validate the candidate by executing the parachain Runtime.
+Collators are similar to validators on any other blockchain. Still, they do not need to provide security
+guarantees because the relay chain provides those as a service to them via [coretime](./learn-agile-coretime.md). If a parachain block is invalid, it will get
+rejected by validators.
 
-Once a candidate meets a specified criteria for inclusion, the selected relay chain block author
-then chooses any of the backable candidates for each parachain and includes those into the relay
-chain block. We say the candidate blocks are _backed_.
+### Rollup Censorship
 
-The assumption that having more collators is better or more secure is not correct. On the contrary,
-too many collators may slow down the network. The only nefarious power collators have is transaction
-censorship. To prevent censorship, a parachain only needs to ensure that there are some neutral
-collators - but not necessarily a majority. Theoretically, the censorship problem is solved by
+The assumption that having more collators is better or more secure is incorrect. On the contrary, too many collators may slow down the network. The only power collators have is transaction censorship. To prevent censorship, a parachain only needs to ensure the collators set is [sufficiently decentralized](../general/web3-and-polkadot.md#decentralization) and that there are some neutral
```

Comment:
the sentence "more collators is more secure is incorrect ...." and "only power collators..." can be removed. Rather what we want to emphasise is: rollup safety (i.e. invalid transaction are not included on the rollup) is ensured by the relay chain using ELVES. But the onus of liveness (i.e. new blocks are eventually produced) and censorship resistance on the rollup is completely on the collators. Hence, a sufficiently decentralised collator set ensures liveness and mitigates censorship resistance. In theory, a single honest collator can ensure liveness but with reduced throughput.